### ISSUE TITLE
Error with Rails 5.2.0: undefined method `respond_to'

### ISSUE THREAD

**Issue by nileshtrivedi** [madzhuga/rails_workflow#40]
```
undefined method `respond_to' for RailsWorkflow::OperationsController:Class Did you mean? respond_to?
```

**Comment by madzhuga** [comment]
Hi @nileshtrivedi ! Yes, you right.

**Comment by nileshtrivedi** [comment]
```
undefined method `respond_to' for RailsWorkflow::OperationsController:Class Did you mean? respond_to?
rails_workflow (0.7.2) app/controllers/rails_workflow/operations_controller.rb:8:in `<class:OperationsController>'
rails_workflow (0.7.2) app/controllers/rails_workflow/operations_controller.rb:6:in `<module:RailsWorkflow>'
rails_workflow (0.7.2) app/controllers/rails_workflow/operations_controller.rb:3:in `<main>'
bootsnap (1.3.0) lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:100:in `load'
bootsnap (1.3.0) lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:100:in `load'
activesupport (5.2.0) lib/active_support/dependencies.rb:468:in `block in load_file'
activesupport (5.2.0) lib/active_support/dependencies.rb:653:in `new_constants_in'
activesupport (5.2.0) lib/active_support/dependencies.rb:467:in `load_file'
activesupport (5.2.0) lib/active_support/dependencies.rb:365:in `block in require_or_load'
activesupport (5.2.0) lib/active_support/dependencies.rb:37:in `block in load_interlock'
activesupport (5.2.0) lib/active_support/dependencies/interlock.rb:14:in `block in loading'
activesupport (5.2.0) lib/active_support/concurrency/share_lock.rb:151:in `exclusive'
activesupport (5.2.0) lib/active_support/dependencies/interlock.rb:13:in `loading'
activesupport (5.2.0) lib/active_support/dependencies.rb:37:in `load_interlock'
activesupport (5.2.0) lib/active_support/dependencies.rb:348:in `require_or_load'
activesupport (5.2.0) lib/active_support/dependencies.rb:502:in `load_missing_constant'
bootsnap (1.3.0) lib/bootsnap/load_path_cache/core_ext/active_support.rb:43:in `load_missing_constant'
activesupport (5.2.0) lib/active_support/dependencies.rb:193:in `const_missing'
activesupport (5.2.0) lib/active_support/inflector/methods.rb:285:in `const_get'
activesupport (5.2.0) lib/active_support/inflector/methods.rb:285:in `block in constantize'
activesupport (5.2.0) lib/active_support/inflector/methods.rb:281:in `each'
activesupport (5.2.0) lib/active_support/inflector/methods.rb:281:in `inject'
activesupport (5.2.0) lib/active_support/inflector/methods.rb:281:in `constantize'
activesupport (5.2.0) lib/active_support/dependencies.rb:574:in `get'
activesupport (5.2.0) lib/active_support/dependencies.rb:605:in `constantize'
actionpack (5.2.0) lib/action_dispatch/http/request.rb:88:in `controller_class_for'
actionpack (5.2.0) lib/action_dispatch/http/request.rb:81:in `controller_class'
actionpack (5.2.0) lib/action_dispatch/routing/route_set.rb:46:in `controller'
actionpack (5.2.0) lib/action_dispatch/routing/route_set.rb:32:in `serve'
actionpack (5.2.0) lib/action_dispatch/journey/router.rb:52:in `block in serve'
actionpack (5.2.0) lib/action_dispatch/journey/router.rb:35:in `each'
actionpack (5.2.0) lib/action_dispatch/journey/router.rb:35:in `serve'
actionpack (5.2.0) lib/action_dispatch/routing/route_set.rb:840:in `call'
railties (5.2.0) lib/rails/engine.rb:524:in `call'
railties (5.2.0) lib/rails/railtie.rb:190:in `public_send'
railties (5.2.0) lib/rails/railtie.rb:190:in `method_missing'
actionpack (5.2.0) lib/action_dispatch/routing/mapper.rb:19:in `block in <class:Constraints>'
actionpack (5.2.0) lib/action_dispatch/routing/mapper.rb:48:in `serve'
actionpack (5.2.0) lib/action_dispatch/journey/router.rb:52:in `block in serve'
actionpack (5.2.0) lib/action_dispatch/journey/router.rb:35:in `each'
actionpack (5.2.0) lib/action_dispatch/journey/router.rb:35:in `serve'
actionpack (5.2.0) lib/action_dispatch/routing/route_set.rb:840:in `call'
rack (2.0.5) lib/rack/tempfile_reaper.rb:15:in `call'
rack (2.0.5) lib/rack/etag.rb:25:in `call'
rack (2.0.5) lib/rack/conditional_get.rb:25:in `call'
rack (2.0.5) lib/rack/head.rb:12:in `call'
actionpack (5.2.0) lib/action_dispatch/http/content_security_policy.rb:18:in `call'
rack (2.0.5) lib/rack/session/abstract/id.rb:232:in `context'
rack (2.0.5) lib/rack/session/abstract/id.rb:226:in `call'
actionpack (5.2.0) lib/action_dispatch/middleware/cookies.rb:670:in `call'
activerecord (5.2.0) lib/active_record/migration.rb:559:in `call'
actionpack (5.2.0) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
activesupport (5.2.0) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (5.2.0) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (5.2.0) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (5.2.0) lib/action_dispatch/middleware/debug_exceptions.rb:61:in `call'
web-console (3.6.2) lib/web_console/middleware.rb:135:in `call_app'
web-console (3.6.2) lib/web_console/middleware.rb:30:in `block in call'
web-console (3.6.2) lib/web_console/middleware.rb:20:in `catch'
web-console (3.6.2) lib/web_console/middleware.rb:20:in `call'
actionpack (5.2.0) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (5.2.0) lib/rails/rack/logger.rb:38:in `call_app'
railties (5.2.0) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (5.2.0) lib/active_support/tagged_logging.rb:71:in `block in tagged'
activesupport (5.2.0) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (5.2.0) lib/active_support/tagged_logging.rb:71:in `tagged'
railties (5.2.0) lib/rails/rack/logger.rb:26:in `call'
sprockets-rails (3.2.1) lib/sprockets/rails/quiet_assets.rb:13:in `call'
actionpack (5.2.0) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
request_store (1.4.1) lib/request_store/middleware.rb:19:in `call'
actionpack (5.2.0) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.0.5) lib/rack/method_override.rb:22:in `call'
rack (2.0.5) lib/rack/runtime.rb:22:in `call'
activesupport (5.2.0) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (5.2.0) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (5.2.0) lib/action_dispatch/middleware/static.rb:127:in `call'
rack (2.0.5) lib/rack/sendfile.rb:111:in `call'
railties (5.2.0) lib/rails/engine.rb:524:in `call'
puma (3.11.4) lib/puma/configuration.rb:225:in `call'
puma (3.11.4) lib/puma/server.rb:632:in `handle_request'
puma (3.11.4) lib/puma/server.rb:446:in `process_client'
puma (3.11.4) lib/puma/server.rb:306:in `block in run'
puma (3.11.4) lib/puma/thread_pool.rb:120:in `block in spawn_thread'
```

**Comment by madzhuga** [comment]
@nileshtrivedi it was already fixed in master, I just published the new version of a gem 0.7.3. with this issue fix.
